reflex/components/graphing/recharts/recharts.py

```diff
@@ -0,0 +1,21 @@
+"""A component that wraps a recharts lib."""
+
+from typing import List
+
+from reflex.components.component import Component, NoSSRComponent
+
+
+class Recharts(Component):
+    """A component that wraps a victory lib."""
+
+    library = "recharts"
+
+    lib_dependencies: List[str] = ["recharts@^2.8.0"]
```

Comment:
```suggestion
    library = "recharts@^2.8.0"
```

does this work? if so i think that's preferred